examples/online.rs

```diff
@@ -2,14 +2,50 @@ pub use vrchatapi::apis;
 
 #[tokio::main]
 async fn main() {
-    let mut config = apis::configuration::Configuration::default();
-    config.basic_auth = Some((String::from("username"), Some(String::from("password"))));
+    let config = apis::configuration::Configuration {
+        // credentials do not belong in source code, use arguments or environment variables
+        basic_auth: Some((String::from("username"), Some(String::from("password")))),
+        ..Default::default()
+    };
 
-    match apis::authentication_api::get_current_user(&config).await.unwrap() {
-        vrchatapi::models::EitherUserOrTwoFactor::CurrentUser(me) => println!("Username: {}", me.username.unwrap()),
-        vrchatapi::models::EitherUserOrTwoFactor::RequiresTwoFactorAuth(requires_auth) => println!("The Username requires Auth: {:?}", requires_auth.requires_two_factor_auth)
-    }
+    let current_user = match apis::authentication_api::get_current_user(&config)
+        .await
+        .expect("Failed to get current user")
+    {
+        vrchatapi::models::GetCurrentUser200Response::CurrentUser(user) => user,
+        vrchatapi::models::GetCurrentUser200Response::TwoFactorRequired(_two_factor_resp) => {
+            // The API returns you a list of methods to verify your 2FA, this determines what function to call next
+            // this example assumes you have a TOTP code
 
-    let online = apis::system_api::get_current_online_users(&config).await.unwrap();
+            let code = "123456"; // grab this via stdin or generate it
+
+            let resp = apis::authentication_api::verify2_fa(
+                &config,
+                vrchatapi::models::TwoFactorAuthCode {
+                    code: code.to_string(),
+                },
+            )
+            .await
+            .expect("Failed to get 2FA response");
+
+            if !resp.verified {
+                panic!("Failed to verify 2FA");
+            }
```

Comment:
we should do things properly in the example.
Imo this includes handling the different 2fa methods well.

Reply:
Fixed [here](https://github.com/vrchatapi/vrchatapi-rust/pull/17/commits/c77b0f795b5558473e5415615d5f43538ed4e87c)

Reply:
This can be marked as resolved (I can't do it myself)